src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -41,17 +46,22 @@
         public static final TrajectoryConstraint[] TRAJECTORY_CONSTRAINTS = {
                 new FeedforwardConstraint(3.0, FEEDFORWARD_CONSTANTS.getVelocityConstant(),
                         FEEDFORWARD_CONSTANTS.getAccelerationConstant(), false), // old value was 11.0
-                new MaxAccelerationConstraint(3.0), // old value was 12.5 * 12.0
-                new CentripetalAccelerationConstraint(3.0) // old value was 15 * 12.0
+                new MaxAccelerationConstraint(1.0), // old value was 12.5 * 12.0
+                new CentripetalAccelerationConstraint(3.0), // old value was 15 * 12.0
+                // in inches
+                new FeedforwardConstraint(11.0, FEEDFORWARD_CONSTANTS.getVelocityConstant(),
+                        FEEDFORWARD_CONSTANTS.getAccelerationConstant(), false),
+                new MaxAccelerationConstraint(1),
+                new CentripetalAccelerationConstraint(15 * 12.0)
```

Comment:
Still duplicating constraints.  And since these aren't used by the WPI commands, I'd recommend leaving the constants as-is so the Jack-in-the-bot path code doesn't change

Reply:
ok i changed back to 3